### PR TITLE
Compilation of site updates 2018-07-12

### PR DESCRIPTION
**Pull request by SMBuilt**
Some requested changes

Board and team member bios updated with what has been received so far
Setup fee for large churches removed
Search menu reworking to make it more in line with top navigation
Old training event replaced by placeholder information
Philosophy + Tools merged in footer and link to #how as Pitch added

Let me know if there are any issues with the changes.

### CODE REVIEW

File: src/components/featured/feature-who.js [hunk 1]
```diff
@@ -47,8 +47,8 @@ const FeaturedWho = () => (
 		</p>
 
 		<div className="expand">
-			<Link to="/about#team">
-				<p>Meet the team</p>
+			<Link to="/about">
+				<p>Meet the team and read our story</p>
```
Comment (BarryThePenguin) [inline src/components/featured/feature-who.js:51]: Yes! 👍

File: src/components/search-bar/index.js [hunk 2]
```diff
@@ -111,18 +111,42 @@ class SearchBar extends React.Component {
...
 							</li>
+							<li>
+								<Link to="/feature">Feature Request</Link>
+							</li>
 							<li>
 								<Link to="/status">Status</Link>
 							</li>
 							<li>
-								<Link to="/documentation">Documentation</Link>
+								<Link to="/documentation">Help + How-to</Link>
```
Comment (BarryThePenguin) [inline src/components/search-bar/index.js:149]: More discoverable links is a 👍 

Thoughts @readeral?

**Review by BarryThePenguin — APPROVED** [review]
👍 looks good

File: src/components/search-bar/index.js (hunk 2 above)
Comment (readeral) [reply src/components/search-bar/index.js:149]: Yeah I agree. Anything to get people using the documentation more! Haha..

File: src/components/featured/feature-who.js (hunk 1 above)
Comment (readeral) [reply src/components/featured/feature-who.js:51]: Nice

**Pull request merged**